### ISSUE TITLE
Update clang-format

### ISSUE THREAD

**Issue by streeve** [ECP-copa/Cabana#530]
CI is failing with clang-14 differences in formatting versus our required clang-10. Maybe it's time to update to at least 11

Any preferences?

**Comment by junghans** [comment]
No preference.

**Comment by dalg24** [comment]
Why is the version not pinned in the CI?
With respect to updating, I am all for it.  I would go for as recent of a version that we think we can afford.  Why do you suggest 11?

**Comment by streeve** [comment]
You're right - we need a pinned version, but it was never previously an issue. Ubuntu focal does not have clang-format-10, so updating now will be easiest. I'm open to suggestions on how new a required version is reasonable - I only said 11 as the most conservative

**Comment by streeve** [comment]
Now that I got it passing, @dalg24 do you think 11 is too old? We can choose newer
